test/test.root.hits

```diff
@@ -0,0 +1,1181 @@
+624 119 3
```

Comment:
I guess this file still has to be taken out (now in legend-testdata), right?

Reply:
Yeah this file can be moved elsewhere if you don't want the test data here.

Reply:
Yes, please - we just did merge it into legend-testdata, after all. :-) I'll see that legend-testdata and LegendTestData.jl get a proper master branch over the weekend.

Reply:
I think the one in legend testdata might actually be different to this one if you merged my earlier PR. I'll check and make a new one if needed.

Reply:
Sure, whichever file you think best, as long as it isn't too big.